lib/bindings/bmap.c

```diff
@@ -185,7 +184,7 @@ void bmap_free(bmap_allocator_t *alloc, void *addr, size_t n)
      * Verify that:
      *    addr is page-aligned
      *    addr is within the range given to alloc
-     *    n is at least 1; the maxiumum size of n is checked in clearn_at().
+     *    n is at least 1; the maximum size of n is checked in clearn_at().
```

Comment:
```suggestion
     *    n is at least 1; the maximum size of n is checked in setn_at().
```

the function called below is `setn_at`, not `clearn_at`.